api/v1/policy_types.go

```diff
@@ -108,13 +108,26 @@ type DetailsPerTemplate struct {
 	History         []ComplianceHistory `json:"history,omitempty"`
 }
 
+// ReplicatedDetailsPerTemplate defines the replicated policy compliance details and history
+type ReplicatedDetailsPerTemplate struct {
+	// +kubebuilder:pruning:PreserveUnknownFields
+	ComplianceState ComplianceState               `json:"compliant,omitempty"`
+	History         []ReplicatedComplianceHistory `json:"history,omitempty"`
+}
+
 // ComplianceHistory defines compliance details history
 type ComplianceHistory struct {
 	LastTimestamp metav1.Time `json:"lastTimestamp,omitempty" protobuf:"bytes,7,opt,name=lastTimestamp"`
 	Message       string      `json:"message,omitempty" protobuf:"bytes,4,opt,name=message"`
 	EventName     string      `json:"eventName,omitempty"`
 }
 
+// ReplicatedComplianceHistory defines the replicated policy compliance details history
+type ReplicatedComplianceHistory struct {
```

Comment:
Could you please move this to api/v1beta1/policyautomation_types.go since this replicated policies actually do use `ComplianceHistory`?